src/thirdParty/registerItem.js

```diff
@@ -15,19 +20,22 @@ import { lib } from '../lib.js';
  * @returns {void}
  */
 export default function(type, name, item, overwrite = false) {
-  if (isItemNameRegistered(type, name)) {
-    console.warn(`${type}/${name} is already registered`);
+  const itemKey = `${type}/${name}`;
+  const alreadyRegistered = isItemNameRegistered(itemKey);
+
+  if (alreadyRegistered && !overwrite) {
+    log('%s is already registered', itemKey);
+
+    return;
+  }
 
-    if (overwrite) {
-      console.warn(`Overwriting ${type}/${name}`);
-    } else {
-      return;
-    }
+  if (alreadyRegistered) {
+    log('Overwriting %s', itemKey);
   }
```

Comment:
As I'm going through files, if I find a pattern that feels odd to me I'm doing minor refactors. This is my OCD coming out. If you'd rather I didn't do this please let me know. No hard feelings 😉

Reply:
It's all good, I wrote the whole plugin system in about 2 days during the 3.0 hype, any refactoring is very welcome!